src/AISmart.GAgent.Twitter/Agent/TwitterGAgentState.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using AISmart.Agent.GEvents;
+using AISmart.Agents;
+using Orleans;
+
+namespace AISmart.Agent;
+
+public class TwitterGAgentState : StateBase
+{
+    [Id(0)] public Guid Id { get; set; } = Guid.NewGuid();
+    [Id(1)] public string UserId { get; set; }
+    [Id(2)] public string Token { get; set; }
+    [Id(3)] public string TokenSecret { get; set; }
```

Comment:
This should not be stored in this state because this state will be saved into cqrs for read. You should create a grain persistent memory to store tokensecret and token